assets/js/modules/analytics-4/datastore/properties.test.js

```diff
@@ -1308,6 +1436,103 @@ describe( 'modules/analytics-4 properties', () => {
 			} );
 		} );
 
+		describe( 'getPropertyCreateTime', () => {
+			it( 'should use a resolver to make a network request', async () => {
+				fetchMock.get( propertyEndpoint, {
+					body: fixtures.properties[ 0 ],
+					status: 200,
+				} );
+
+				const propertyID = '12345';
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.setPropertyID( propertyID );
+
+				const initalPropertyCreateTime = registry
+					.select( MODULES_ANALYTICS_4 )
+					.getPropertyCreateTime();
+
+				expect( initalPropertyCreateTime ).toBeUndefined();
+
+				await untilResolved(
+					registry,
+					MODULES_ANALYTICS_4
+				).getPropertyCreateTime();
+				expect( fetchMock ).toHaveFetched( propertyEndpoint, {
+					query: { propertyID },
+				} );
+
+				const propertyCreateTime = registry
+					.select( MODULES_ANALYTICS_4 )
+					.getPropertyCreateTime();
+
+				expect( propertyCreateTime ).toEqual(
+					new Date( fixtures.properties[ 0 ].createTime ).getTime()
+				);
+				expect( fetchMock ).toHaveFetchedTimes( 1 );
+			} );
+
+			it( 'should reset create time when new property is created', async () => {
+				provideUserAuthentication( registry );
+
+				const accountID = fixtures.createProperty.parent;
+
+				fetchMock.post( createPropertyEndpoint, {
+					body: fixtures.createProperty,
+					status: 200,
+				} );
+
+				muteFetch( propertyEndpoint );
+
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.selectProperty( PROPERTY_CREATE );
```

Comment:
@aaemnnosttv I simulated the full creation of property flow, by setting `CREATE_PROPERTY`, so test can be correctly asserting that property create time will be reset to default `0` in this scenario